mne/bem.py

```diff
@@ -1667,18 +1701,22 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
     outer skin) from multiecho FLASH MRI data with spin angles of 5 and 30
     degrees, in mgz format.
 
-    This function assumes that the flash images are available in the
-    folder mri/bem/flash within the freesurfer subject reconstruction.
-
     See Also
     --------
     convert_flash_mris
     """
     from .viz.misc import plot_bem
+    from .surface import write_surface, _load_ascii_surface
+
+    # Hack to enable testing with travis.
+    is_test = os.environ.get('FREESURFER_HOME', '').endswith(
+        'MNE-testing-data')
```

Comment:
This is just for skipping the parts using freesurfer functions on travis. I think it's better than not testing at all.